matter/efr32/efr32mg24/BRD4186C/autogen/sl_component_catalog.h

```diff
@@ -15,6 +14,33 @@
 #define SL_CATALOG_BLUETOOTH_RTOS_ADAPTATION_PRESENT
 #define SL_CATALOG_BLUETOOTH_PRESENT
 #define SL_CATALOG_BLUETOOTH_FEATURE_MULTIPROTOCOL_PRESENT
+#ifdef CHIP_917
+#define SL_CATALOG_MATTER_ACCESS_CONTROL_PRESENT
```

Comment:
Why are those needed ? those all all define tie to slc matter component for matter clusters. Which doesn't do anything in CSA SMG context. 

Is there another usage for those?

Reply:
removed